definitions/video-datazoom_video/golden_metrics.yml

```diff
@@ -0,0 +1,30 @@
+videoStartTime:
```

Comment:
these golden metrics look very similar in concept to existing video players entities.

What is the reason to have multiple types in this case?

Example: https://github.com/newrelic/entity-definitions/blob/main/definitions/video-browser_video/golden_metrics.yml